contrib/debian/control

```diff
@@ -2,8 +2,7 @@ Source: aziot-identity-service
 Section: admin
 Priority: optional
 Maintainer: Azure IoT Edge Devs
-Build-Depends: debhelper (>=9),
-               dh-systemd
+Build-Depends: debhelper (>=9.20160709)
```

Comment:
Why that specific version?

Reply:
It's the one mentioned in the bug I linked in the description.